CONTRIBUTING.md

```diff
@@ -0,0 +1,35 @@
+## Contributing
+
+Post bug reports, feature requests, and questions in [Issues](https://github.com/microsoft/fluentui-apple/issues).
+
+This project welcomes contributions and suggestions.  Most contributions require you to agree to a
+Contributor License Agreement (CLA) declaring that you have the right to, and actually do, grant us
+the rights to use your contribution. For details, visit https://cla.opensource.microsoft.com.
+
+When you submit a pull request, a CLA bot will automatically determine whether you need to provide
+a CLA and decorate the PR appropriately (e.g., status check, comment). Simply follow the instructions
+provided by the bot. You will only need to do this once across all repos using our CLA.
+
+This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/).
+For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or
+contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.
+
+### Prerequisite
+#### Xcode
+Make sure your local [Xcode version](https://github.com/microsoft/fluentui-apple#requirements) is the same as the supported version of the repo. 
+
+#### Swift Lint
+This project uses [SwiftLint](https://github.com/realm/SwiftLint) to automatically lint our Swift code for common errors. Please install it when developing in this repo by following the [SwiftLint Installation Instructions](https://realm.github.io/SwiftLint/).
+
+### Developing in the repo
+
+Fluent UI Apple requires all [pull requests](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/about-pull-requests) to come from forks of the repository. Please see [Fork a Repo - GitHub Help](https://help.github.com/en/github/getting-started-with-github/fork-a-repo) for more details on how to set up a fork of Microsoft/fluentui-apple, keep it up-to-date, and submit pull requests back to this repository.
+
+Fluent UI Apple doesn't have any external code dependencies, so developing in the repository is as easy as launching the appropriate Xcode project or workspace and building and running a test/demo app.
```

Comment:
Is it worth it to specify what "the appropriate project/workspace" is, or how to run the demo app? If we already have that outlined somewhere, we should link it here.

Reply:
Ah, reading further down, we DO specify. I think we should re-word this to say something like (but better than) "instructions to run the demo app are in the respective CONTRIBUTING.MD files for ios and macos" with links to those files.

Reply:
hmmm i don't want it to be too wordy but i reword the next sentence a bit